tests/test_sum.py

```diff
@@ -0,0 +1,45 @@
+import jax_galsim as galsim
+
+
+def test_gsparams():
+    """Test withGSParams with some non-default gsparams"""
+    obj1 = galsim.Exponential(half_light_radius=1.7)
+    obj2 = galsim.Gaussian(sigma=0.2)
+    gsp = galsim.GSParams(
+        folding_threshold=1.0e-4, maxk_threshold=1.0e-4, maximum_fft_size=1.0e4
+    )
+    gsp2 = galsim.GSParams(folding_threshold=1.0e-2, maxk_threshold=1.0e-2)
+
+    sum = galsim.Sum(obj1, obj2)
+    sum1 = sum.withGSParams(gsp)
+    assert sum.gsparams == galsim.GSParams()
+    assert sum1.gsparams == gsp
+    assert sum1.obj_list[0].gsparams == gsp
+    assert sum1.obj_list[1].gsparams == gsp
+
+    sum2 = galsim.Sum(obj1.withGSParams(gsp), obj2.withGSParams(gsp))
+    sum3 = galsim.Sum(
+        galsim.Exponential(half_light_radius=1.7, gsparams=gsp),
+        galsim.Gaussian(sigma=0.2),
+    )
+    sum4 = galsim.Add(obj1, obj2, gsparams=gsp)
```

Comment:
you might also just want to check `obj1 + obj2` also works?

Reply:
I see, that makes sense. As I was reviewing I kept forgetting the extent to which you copy-pasted the tests (which I think you commented on below)